### PR TITLE
Add information about the case when certificates are issued by an CA

### PR DESCRIPTION
**Pull request by phozzy**
This PR adds information about the case when certificates are issued by an CA. Describes what is fullchain certificate and how-to use it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +116,51 @@ openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes \
...
+
+CA usually provide you with 4 files of following types:
+
+* certificate file for your hostname (eg. `cert.pem`);
+
+* private key file for your hostname (eg. `privkey.pem`);
+
+* chain file (eg. `chain.pem`) - this file contains a certificate chain of CA
+
+```bash
```
Comment (gijskant) [inline README.md]: Maybe indent this part, so that is is part of the 'chain file' bullet

**Review by gijskant — APPROVED** [review]
Nice, very readable explanation!

File: README.md [hunk 2]
```diff
@@ -107,7 +116,51 @@ openssl req -new -newkey rsa:4096 -x509 -sha256 -days 365 -nodes \
...
+-----BEGIN CERTIFICATE-----
+  Certificate of Root CA
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+  Certificate of CA(1) signed by Root CA
+-----END CERTIFICATE-----
+...
+-----BEGIN CERTIFICATE-----
+  Certificate of CA(n) signed by CA(n-1)
+-----END CERTIFICATE-----
```
Comment (ewelinajozwik) [inline README.md]: didn't it require a different order? First `CA(n) signed by CA(n-1)` then `Certificate of Root CA` then `Certificate of CA(1) signed by Root CA`? I think it was like this for the fix we added.

Comment (phozzy) [reply README.md]: No, I doulble checked with client installation it should be in thist order. Quite not consistently, but it is this way: your certificate -> root ca -> ca1 signed by root ca -> ca2 signed by ca1 -> ... -> ca your certificate signed by. ¯\_(ツ)_/¯
initially we put our certificate at the end, and that was an issue.

File: README.md (hunk 1 above)
Comment (phozzy) [reply README.md]: Fixed.

**Review by ewelinajozwik — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ewelinajozwik) [reply README.md]: ok, now it's clear to me, thanks!

**Pull request merged**